app/views/templates/_form.html.erb

```diff
@@ -37,7 +37,8 @@
       <% else -%>
         <%= alert :class => 'alert-success', :header => '',
                   :text  => icon_text("info-sign", (_('Note: %s ') % link_to(_('Useful template functions and macros'),
-                                                                             "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#4.4.3ProvisioningTemplates", :rel => 'external')).html_safe) %>
+                                                                             type == 'ptable' ? "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#4.4.4PartitionTables" : "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#4.4.3ProvisioningTemplates",
```

Comment:
Also, there are probably 5-6 places with the url `"http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html` maybe put it in a helper so all places can use it (it may be another PR, though)